src/drivers/dw/dma.c

```diff
@@ -65,7 +65,7 @@
 #include <platform/interrupt.h>
 #include <errno.h>
 #include <stdint.h>
-#include <string.h>
+#include <sof/string.h>
```

Comment:
Can we have *fixes* like this in first commit? Then *changes* that rely on it in next one? (But in the 1 PR)

Reply:
No. This change is a bugfix and prework in one i wouldn't connect it in one pr. I prepared it so every change is self contained.